infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/pvc/PerWorkspacePVCStrategyTest.java

```diff
@@ -113,6 +115,7 @@ public void shouldReturnPVCPerWorkspace() throws Exception {
     // then
     assertEquals(commonPVC.getMetadata().getName(), PVC_NAME_PREFIX + "-" + WORKSPACE_ID);
     assertEquals(commonPVC.getMetadata().getLabels().get(CHE_WORKSPACE_ID_LABEL), WORKSPACE_ID);
+    assertEquals(commonPVC.getSpec().getStorageClassName(), PVC_STORAGE_CLASS_NAME);
```

Comment:
Consider adding test cases to assert the default behavior here as well.

Reply:
I added the test case. (Please refer my comment with the mention to you)